scripts/install-binary.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+PREFIX=${PREFIX:-/usr/local}
```

Comment:
Instead of an ENV can we just make this an optional argument
```
sh install-binary.sh /usr/local
```

Reply:
Shell arguments are always mystery for new user. `PREFIX=/usr sh install.sh` is more explicit than `sh install.sh /usr`. I prefer ENV style. Also in future we might add new arguments for  something else.
So for now, how about this?
```
PREFIX=${PREFIX:-${1:-/usr/local}}
```

Reply:
That's fine 👍 
I suspect most people won't specify the path anyway.
Can we also rename the script simply `install.sh`? Shorter to type and it's installing more than just a binary

Reply:
Yep, I already rename it as `install.sh` in `archive.sh`, but will simply rename it, removing unzipping part. (as we discussed in slack) 👍

Reply:
Fixed in c0aabd0